editors/nvi/files/patch-cl_cl.h.diff

```diff
@@ -0,0 +1,17 @@
+--- cl/cl.h.orig	2023-09-22 20:32:31.000000000 -0700
++++ cl/cl.h	2023-09-23 00:07:27.000000000 -0700
+@@ -20,8 +20,14 @@
+ #else /* } { */
+ #ifdef HAVE_NCURSES_H /* { */
+ #include <ncurses.h>
++#include <term.h>
++#define HAVE_CURSES_SETUPTERM 1
++#define HAVE_CURSES_NEWTERM 1
+ #else /* } { */
+ #include <curses.h>
++#include <term.h>
++#define HAVE_CURSES_SETUPTERM 1
++#define HAVE_CURSES_NEWTERM 1
```

Comment:
So in the `if` part, you include a file and define two macros, and in the `else` part you do exactly the same thing? Why not do it just one time, outside of the `if` statement?

Which bug does this patch fix? It's a good idea to write, at the top of the patchfile, what purpose the patch serves. I like to list the actual error message that it fixes.

Where did the patch come from? Put the URL where you got it (or, if you wrote it yourself, the upstream bug report or pull request where you submitted it to them) at the top of the patchfile too.

Reply:
Sorry this was a mistake.  The new lines are needed only for ncurses and are needed to fix a missing prototype error.  I have made a new patch for this and noted the reason in the preamble.  Hopefully, I have done it correctly.